src/raven_logger_backend.erl

```diff
@@ -4,15 +4,43 @@
 
 -define(META_FILTER, [gl,pid,time,file,line,mfa,span_ctx]).
 
+%% API
+
 log(LogEvent, _Config) ->
-	case is_httpc_log(LogEvent) of
+	try log(LogEvent)
+	catch _:Reason:StackTrace ->
+		LE = list_to_binary(lists:flatten(io_lib:format("~0p", [LogEvent]))),
+		ST = list_to_binary(lists:flatten(io_lib:format("~0p", [StackTrace]))),
+		logger:warning(#{message => <<"Raven logger backend crashed">>,
```

Comment:
use the LOG_WARNING macro from the logger.hrl instead, it adds metadata mfa, file and line.

Reply:
OK